### ISSUE TITLE
Change list items backgrounds

### ISSUE THREAD

**Issue by Nickfrm** [rigami/readme#11]
It would be awesome if you can implement such feature like "changing backgrounds on list items". Here's example:

![image](https://user-images.githubusercontent.com/35500726/122763600-2fe76f80-d2a7-11eb-8fe6-0ab31d1a1275.png)

**Comment by danilkinkin** [comment]
Hello, sorry for late answer. Now i'm migrate to new extensions API (MV3).

Do you want to set this color yourself? Or I do not quite understand under what conditions it should change. Can you tell us in more detail why this is needed?

Might consider adjusting the accent of the whole application. But not this time, many tasks and little time

**Comment by danilkinkin** [comment]
I`m added this idea in todos. Later back to this

**Comment by Nickfrm** [comment]
Yes, I suggest changing color by yourself. It can be really useful for people like me who have a lot of folders. And the possibility of changing its backgrounds will make it much easier to find the needed ones. I know we can add folders to Favourites but my suggestion will be like an extended version of this. See how groups work in Chrome, I think about something similar.

**Comment by danilkinkin** [comment]
What about emojis icons? They are bright and can help you navigate.

![Group 340](https://user-images.githubusercontent.com/10831688/127970849-3218d75d-e723-4013-94b0-6df7dc145dee.png)

**Comment by Nickfrm** [comment]
In my opinion, backgrounds look more UX friendly but this also makes sense
